dashboard/app/models/concerns/concept_difficulties.rb

```diff
@@ -0,0 +1,23 @@
+module ConceptDifficulties
+  extend ActiveSupport::Concern
```

Comment:
It almost certainly could; I made it a concern because I was originally thinking about doing more with it than I ended up doing.